helm/kagent/templates/deployment.yaml

```diff
@@ -60,14 +60,6 @@ spec:
           image: "{{ .Values.app.image.registry }}/{{ .Values.app.image.repository }}:{{ coalesce .Values.global.tag .Values.app.image.tag .Chart.Version }}"
           imagePullPolicy: {{ .Values.app.image.pullPolicy }}
           env:
-            {{- $model := index .Values.providers .Values.providers.default  }}
-            {{- if and (ne $model.apiKeySecretRef "") $model.enabled }}
-            - name: {{ $model.apiKeySecretKey }}
```

Comment:
don't we still need to set the key here?

Reply:
I don't think we do anymore? Right now it's NEVER being added anyway, I think we should wait for Scott's PR which changes how the docs tool adds the key.

Reply:
yes! I'd wait for his PR to go in first, that way we're 100% that the doc tool will continue working